### ISSUE TITLE
Add terminology/references/dictionary

### ISSUE THREAD

**Issue by hzoo** [babel/babel.github.io#1120]
Add a page to explain Babel/js specific terms that users might not know at first

like: es6/es2015, TC39, plugin, preset, etc.

Not sure how we want to do this but at least we can come up with some terms, and then how to show it on the site and whether we should be linking to it in some way.

**Comment by brigand** [comment]
One option is to show the definition on sustained hover. It could use an `<abbr>` tag.

**Comment by nitin42** [comment]
I think these terms are described in brief [here](https://babeljs.io/docs/plugins/). or... not?

**Comment by aaronang** [comment]
@nitin42 Yes you are right, but I think it might be useful to have one central place for common terminology. Easier to find and search 😄

**Comment by nitin42** [comment]
Ok ! I am working on this 😄

**Comment by nitin42** [comment]
Can we also include the different packages? Brief about each package. For eg - `babel-cli`

**Comment by existentialism** [comment]
Maybe the first step is to compile a list of the terms that need defining?

Depending on the number of them, there may be better options than a dedicated page (as @brigand suggested `<abbr />`, or more custom tooltips, etc) as it wouldn't require a user to click back and forth.

**Comment by nitin42** [comment]
![screen shot 2017-03-01 at 9 22 14 pm](https://cloud.githubusercontent.com/assets/16336071/23467898/43f204fa-fec5-11e6-89d1-0965d28f1be7.png)
 Looks great though !

**Comment by existentialism** [comment]
Yeah (though, I _really_ need to get around to improving typography styles on the site)!

Two areas that are worth discussion, IMO:

### Duplication of content

Let's take anywhere that references `ES2015` as an example. Why wouldn't we just link to `Learn ES2015` which contains the same text and way more vs. linking to the term on the `Terminology` page?

I definitely think we want to avoid using the terminology page as a crutch for good information architecture :)

### Linking

It may be outside the initial scope, but I'd like to at least address easing the burden for content creators to remember linking to a term to the terminology page. It might be crude, but something as simple as scanning for and converting "TC39" -> `<abbr title="TC39 is the committee that evolves JavaScript. Its members are companies (among others, all major browser vendors).">TC39</abbr>` might go a long way towards achieving this without needing to remember to link?

**Comment by nitin42** [comment]
Gr8!

**Comment by nitin42** [comment]
So ? What can we do now ? I have a list of terms !
